src/pages/ServiceList/ServiceListPage.tsx

```diff
@@ -126,19 +130,12 @@ class ServiceListPageComponent extends FilterComponent.Component<
       })
       .then(serviceListItems => {
         this.promises.cancel('sort');
-        this.sortItemList(serviceListItems, this.state.currentSortField, this.state.isSortAscending)
-          .then(sorted => {
-            this.setState({
-              listItems: sorted
-            });
-          })
-          .catch(err => {
-            if (!err.isCanceled) {
-              console.debug(err);
-            }
-          });
+        this.setState({
+          listItems: this.sortItemList(serviceListItems, this.state.currentSortField, this.state.isSortAscending)
+        });
       })
       .catch(err => {
+        console.log(err);
```

Comment:
Was this for testing? Can it be removed?

Reply:
Totally :)